tests/core/test_addresses_correctly_assigned.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import time
+
+from django.test import RequestFactory
+
+from pretix_eth.models import WalletAddress, WalletAddressError
+
+hex_addresses = [
+    '0x4257a60613500E8272b0d6151E6CCcAcf79E8Ff4',
+    '0x7D3013fB1Abe40A7bfBAA5E826E0A178F01Ef2F9',
+    '0x4B10a9cbE018B01Ef65B43c622fE0Bd08a8BA160',
+    '0xf5773255D551156182e7F0E3A26bD8d0Bb0022B7',
+    '0xF68b56523CF6ec929953159B67e250a98726349B',
+]
+
+
+@pytest.mark.django_db
+def test_addresses_correctly_assigned(event, provider, get_order_and_payment):
+    provider.settings.set('ETH_RATE', '1.0')
+
+    for address in hex_addresses:
+        WalletAddress.objects.create(event=event, hex_address=address)
```

Comment:
Just going to highlight that you can [`bulk_create`](https://docs.djangoproject.com/en/3.1/ref/models/querysets/#bulk-create) in cases like this.  It's not strictly better.  In this case it might be more cumbersome (I didn't try writing it out..), but I wanted to point it out in case you weren't aware of the API.